test_automation/work_order/test_work_order_invalid_parameter.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import logging
+import work_order_utility
+
+logger = logging.getLogger(__name__)
+
+def test_wo_inv_param(setup_config):
+    """ Testing work order request with invalid workload parameter value. """
+
+    # retrieve values from conftest session fixture
+    worker_obj = setup_config[0]
+    sig_obj = setup_config[1]
+    uri_client = setup_config[2]
+    private_key = setup_config[3]
+    err_cd = setup_config[4]
+
+    # input and output names
+    input_json_file = r'./work_order/workorder_input/wo_inv_param.json'
```

Comment:
A Python `r''` string is only needed if you have backslashes (`\`).  Remove the `r`.

Reply:
Removed the unnecessary raw string in assignment of file name